src/Dhall.hs

```diff
@@ -16,6 +16,7 @@ module Dhall
     (
     -- * Input
       input
+    , rawInput
```

Comment:
My only suggestion is to move this closer to the bottom of the export list under a `-- * Miscellaneous` header before the `-- * Re-exports` section since most people won't be using this function